SABT_MainUnit/audio.c

```diff
@@ -209,16 +209,16 @@ void play_pattern(unsigned char pattern) {
 void play_dot_sequence(glyph_t *this_glyph) {
 	char pattern;
 	if (this_glyph != NULL) {
-		if (this_glyph->parent != NULL) {
+		pattern = this_glyph->pattern;
+		play_pattern(pattern);
+		if (this_glyph->next != NULL) {
 			// Checks for parent glyphs if a last-order pattern
 			sprintf(dbgstr, "[Audio] Playing parent pattern: %s\n\r",

```

Comment:
So now we're playing the "child" pattern (next glyph, right?) So can we change the comment to reflect this?